analyzer/windows/lib/api/process.py

```diff
@@ -244,6 +245,12 @@ def process_thunder(self, sample_path, kernel_pipe, forwarder_pipe, dispatcher_p
 
         log.info("Thunder - post monitor")
 
+        # if package is not office, kill preloaded
+        if package not in PACKAGE_TO_PRELOADED_APPS.keys():
```

Comment:
instead of passing around `package`, how about running this in the initialization of the `Package` class?
the name is available there, so it's cleaner and runs before the actual monitoring